src/serialization/ObjectData.ts

```diff
@@ -124,7 +121,7 @@ export class ObjectDataOutput implements DataOutput {
         const len = (buf != null) ? buf.length : BitsUtil.NULL_ARRAY_LENGTH;
         this.writeInt(len);
         for (let i = 0; i < len; i++) {
-            this.write(buf[i]);
+            this.write((buf as any)[i]);
```

Comment:
The reason for that is the `safe-buffer` polyfill. Its typings don't define index signature, so without `as any` the compiler gives the following error: `TS7017: Element implicitly has an 'any' type because type 'Buffer' has no index signature.`

The `as any` is the most simple workaround for this issue. Once we drop support for older versions of Node, we can remove `safe-buffer` and get rid of such workarounds (there are a couple of them in the code base, so it should be an easy fix).

More on this topic: https://stackoverflow.com/questions/32968332/how-do-i-prevent-the-error-index-signature-of-object-type-implicitly-has-an-an

Reply:
I forgot about the new dependency. I understand it's caused by its lack of typings. I agree it's not a problem then.